assets/cubyz/music/credits.txt

```diff
@@ -1,44 +1,76 @@
-TotalDemented ( https://www.youtube.com/channel/UCut9PCEq4cEJogfEPUXk7Gg ):
-cubyz.ogg
-leaves.ogg
-in_the_depths.ogg
-la_kubisa_caverna.ogg
-la_kubisa_caverna_69.ogg
-tides.ogg
+----------------------------------------------------------------------------------------------------------
+TotalDemented - Disc 1
+youtube.com/@totaldemented
 
+1. Cubyz
+2. Cublajz
+3. Kubis
+4. Vitreous
+5. Scoria
+6. Leaves
+7. In the Depths
+8. Tides
+9. Midnight Melody
+----------------------------------------------------------------------------------------------------------
+Gray Vibes - Disc 2
+youtube.com/@grayvibes2512
+youtube.com/@grayvibes9601
 
-sinanimea:
-sunrise.ogg
-under_the_water_sky.ogg
+1. Mornings
+2. Amb
+----------------------------------------------------------------------------------------------------------
+Bytepop - Disc 3
 
+1. Pixel Wires
+----------------------------------------------------------------------------------------------------------
+Chrythm - Disc 4
+youtube.com/@chrythm
+youtube.com/@chrythmmusic
 
-Mrmayman:
-out_of_breath.ogg
+1. Chrythm Cubyz
+2. Cubyz More
+----------------------------------------------------------------------------------------------------------
+Sinanimea - Disc 5
 
+1. Sunrise
+2. Under the Water Sky
+----------------------------------------------------------------------------------------------------------
+IkabodBTW - Disc 6
+youtube.com/@ikabodBTW
 
+1. Atlantis
+2. Chimes
+----------------------------------------------------------------------------------------------------------
+Mrmayman - Disc 7
 
-Kevin MacLeod
+1. Out Of Breath
+----------------------------------------------------------------------------------------------------------
+Kevin MacLeod - Disc 255
 
-Syncerely by Kevin MacLeod
+1. Static Motion by Kevin MacLeod
+License: https://filmmusic.io/standard-license
```

Comment:
First up, according to the license, a link to the download must be provided.
Secondly it seems that Kevin MacLeod's music is no longer accessible through filmmusic.io anyways, so we should probably look for a more direct source for his music.